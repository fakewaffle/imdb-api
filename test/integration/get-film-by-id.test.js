const expect = require("expect");
const { getFilmById } = require("../../handlers");

describe("getFilmById", () => {
  it("should get a film", async () => {
    const id = "tt1396484"; //It
    const film = await getFilmById(id);

    expect(film).toMatchSnapshot();
  });
});
