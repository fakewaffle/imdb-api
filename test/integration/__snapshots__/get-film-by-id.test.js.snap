// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getFilmById should get a film 1`] = `
Object {
  "length": "2h 15min",
  "rating": "R",
  "title": "It",
  "year": 2017,
}
`;
